File: crates/deslop/tests/fixtures/python-issue-119-role-mismatch/transport.py
@traced
class BoomTransport:
    alpha = 0
    bravo = 0
    charlie = 0
    delta = 0
    echo = 0
    foxtrot = 0
    golf = 0
    hotel = 0
    india = 0
    juliet = 0
    kilo = 0
    lima = 0
    mike = 0
    november = 0


@traced
def test_transport_keep(stub):
    alpha = 0
    bravo = 0
    charlie = 0
    delta = 0
    echo = 0
    foxtrot = 0
    golf = 0
    hotel = 0
    india = 0
    juliet = 0
    kilo = 0
    lima = 0
    mike = 0
    november = 0
    saved.bind(stub)

File: crates/deslop/tests/fixtures/dart-issue-119-role-mismatch/transport.dart
class BoomTransport {
  int alpha = 0;
  int bravo = 0;
  int charlie = 0;
  String delta = "xx";
}

void testTransportKeep(Object stub) {
  final saved = wrap(stub);
  saved.bind(stub);
  assert(saved.count == 0);
}
